tests/testthat/test-create.R

```diff
@@ -1,9 +1,5 @@
 test_that("Create a quarto project", {
   skip_if_no_quarto("1.4")
-  # TODO: Fix the test once issue solve upstream
-  # - https://github.com/quarto-dev/quarto-cli/issues/8809
-  # - https://github.com/quarto-dev/quarto-r/issues/153
-  skip_if_quarto("1.5")
```

Comment:
With my change re: `name`, these tests pass for me with Quarto 1.5, so I think they can be unskipped?